mesh/amqp.h

```diff
@@ -27,9 +28,22 @@ struct mesh_amqp_config {
 	char *routing_key;
 };
 
+struct mesh_amqp_rc_message {
+    uint8_t flags;
+    uint16_t app_idx;
+    uint16_t net_idx;
+    uint8_t element_idx;
+    uint16_t dst_addr;
+    uint8_t ttl;
+    uint16_t data_len;
+    uint8_t data[384];
+} __attribute__((packed));
+
```

Comment:
This should be a part of fd message, the protocol should be usable from the application!

Reply:
Also, what about devkey messages?